gammapy/estimators/map/tests/test_core.py

```diff
@@ -217,8 +217,13 @@ def test_combine_flux_maps(map_flux_estimate, wcs_flux_map, reference_model):
     stop = u.Quantity([1.5, 2.5], "min")
     gti = GTI.create(start, stop)
 
+    data = map_flux_estimate.copy()
+    ts = data["norm"].copy()
+    ts.data = (data["norm"].data / data["norm_err"].data) ** 2
```

Comment:
is it possible that `data["norm_err"].data` is zero?

should you consider also this case?

Reply:
Only if something goes wrong in the estimation.
Added a test for that and now it returns nan instead of inf.